### DISCUSSION TITLE
babel-core running in react js

### DISCUSSION THREAD

**Discussion by Mohammadgeek** [webpack/webpack#17286] [Q&A]
I create a simple project React with Webpack bundle and Babel but Webpack cli error babel-core I uninstall Babel-core or loader and install npm but not fix how to does fix it?

package.json
```
{
  "name": "reactjs",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "build": "webpack"
  },
  "author": "",
  "license": "ISC",
  "devDependencies": {
    "@babel/cli": "^7.13.10",
    "@babel/core": "^7.13.10",
    "@babel/preset-env": "^7.13.10",
    "@babel/preset-react": "^7.13.13",
    "babel-loader": "^7.1.3",
    "css-loader": "^6.7.4",
    "file-loader": "^6.2.0",
    "html-loader": "^0.5.5",
    "html-webpack-plugin": "^4.5.2",
    "react": "^18.2.0",
    "react-dom": "^18.2.0",
    "style-loader": "^2.0.0",
    "webpack": "^5.26.0",
    "webpack-cli": "^4.9.2"
  },
  "dependencies": {}
}
```

error web pack cli 

```
ERROR in ./src/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'babel-core'
```

**Comment by alexander-akait** [comment]
Please use the latest version of `babel-loader` - https://github.com/babel/babel-loader/blob/main/package.json#L3

**Reply by alexander-akait** [reply]
How you generate this template?

**Reply by Mohammadgeek** [reply]
> How you generate this template?

https://github.com/webpack-contrib/css-loader/blob/v5.2.7/package.json#L42

**Reply by Mohammadgeek** [reply]
> Please use the latest version of `babel-loader` - https://github.com/babel/babel-loader/blob/main/package.json#L3

thanks, install latest version but show this error:
```
asset 07258446fbe7dbf211e0.main.js 3.45 KiB [emitted] [immutable] (name: main)
./src/index.js 39 bytes [not cacheable] [built] [code generated] [1 error]

ERROR in ./src/index.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
Error: Cannot find module 'fs/promises'
```
in this error true hash compile js but not cacheable

**Reply by alexander-akait** [reply]
Update your Node.js version

**Reply by Mohammadgeek** [reply]
> Update your Node.js version

why? i used Windows 7 can not upgrade increase 16+;

**Reply by alexander-akait** [reply]
Because you can't use new version of babel using old node.js version, you need to update
